### ISSUE TITLE
Did the System.Windows.Forms assembly version intentionally change in 6.0.2?

### ISSUE THREAD

**Issue by SimonCropp** [dotnet/winforms#6693]
i updated my CI server to sdk 6.0.102 not when i try to run the output i get

```
Unhandled exception. System.IO.FileLoadException: Could not load file or assembly 'System.Windows.Forms, Version=6.0.2.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'. The located assembly's manifest definition does not match the assembly reference. (0x80131040)
File name: 'System.Windows.Forms, Version=6.0.2.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'
   at Program.Main()
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine)
   at Program.Main()
   at Program.<Main>()
```

**Comment by SimonCropp** [comment]
when i moved back to sdk 6.0.101 i get as a ref

```
System.Windows.Forms, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089
```

**Comment by RussKie** [comment]
Duplicate of #6663.

**Comment by RussKie** [comment]
Please refer to https://github.com/dotnet/core/issues/7176 for the workaround.

**Comment by SimonCropp** [comment]
@RussKie thanks. so when this is fixed, my app build against the latest 6.0.* sdk will be able to run on a machine with only 6.0.0 installed?
